explorer/interpreter/type_checker.cpp

```diff
@@ -4556,6 +4567,22 @@ auto TypeChecker::DeclareClassDeclaration(Nonnull<ClassDeclaration*> class_decl,
     class_vtable[fun->name()] = {fun, class_level};
   }
 
+  if (class_decl->extensibility() == ClassExtensibility::None) {
```

Comment:
Could you move that logic to switch case ~ line 4527, in the existing `case`?

Reply:
@Pixep I'm not sure how it can help. That switch is inside `class_decl->members()` loop, but I need to look up inside class vtable if there is any abstract method in chain that is not implemented.
Correct me if I'm not correctly understand this issue. Thanks

Reply:
Sorry for the delay, didn't see the notification. Feel free to ping on Discord anytime!

You're right, then I would recommend having a single function in the type checker to look `abstract` methods in the vtable, which you can use here, and for `case StatementKind::VariableDefinition`, example below